gradle/libs.versions.toml

```diff
@@ -16,6 +17,7 @@ sdk-target = "33"
 timber = "5.0.1"
 
 [libraries]
+androidx-activity-compose = { module = 'androidx.activity:activity-compose', version = '1.7.2' }
```

Comment:
dunno where `setContent` lives except in here. It must be in another dependency too (bc I don't have this dependency explicitly in some other projects) but I dunno which, nor how to easily find out.